### ISSUE TITLE
sus (if statement) and mandelbrot showcase is broken on the latest commit

### ISSUE THREAD

**Issue by etherealxx** [face-hh/bussin#27]
take a look at this code
```
lit x be 0 rn

sus (x fr 0) {
    waffle("x is 0")
}
```
in previous commit 06f0f452fad57d2c33a2736fe8a76f4be3bd7cd3 it will outputs `x is 0`
but on the current latest f616ced84f3365070e09067877f936ced0d5cf82 , it outputs:
```
{ value: 0, type: 'number' } { value: 0, type: 'number' } false true
{ type: 'boolean', value: true }
x is 0
```

mandelbrot example is also broken on the latest commit. i confirmed that it still works on the last commit before current.

**Comment by face-hh** [comment]
I'm aware I left some debug code in the latest release, causing the printing. I'll check what went wrong tomorrow

**Comment by face-hh** [comment]
It has been fixed in v1.2.1, thanks for reporting the bug!
